### PR TITLE
Holium app dev summer

### PR DESCRIPTION
**Pull request by drunkplato**
These are a list of app bounties for our app dev summer initiative. The goal is to have these small apps built by teams of hoon school grads by Assembly.  

We hope to publish the bounties on Thursday, July 21st prior to the scheduled dev call at 10:30am CDT.

### CODE REVIEW

**Review by jalehman — CHANGES_REQUESTED** [review]
These all look great to me so far, but can you remove the application page that was included by accident (this was from your last PR) at `content/applications/~livdel-lomder-librun/ballot.md`?

File: content/grants/app-abacus.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+[extra]
+image = ""
+description = "Abacus: A basic calculator app"
+reward = "1 star ($1k bonus if done by Assembly)"
+assignee = ""
+id = ""
+champion = "Holium"
+completed = false
+work_request_link = ""
```
Comment (mikeosborne) [inline content/grants/app-abacus.md]:
```suggestion
work_request_link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0160&prefill_Grant+Name=Abacus"
```

File: content/grants/app-chronicle.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+[extra]
+image = ""
+description = "Chronicle is a news app that links all articles from group channels"
+reward = "2 stars ($4k bonus if done by Assembly)"
+assignee = ""
+id = ""
+champion = "Holium"
+completed = false
+work_request_link = ""
```
Comment (mikeosborne) [inline content/grants/app-chronicle.md]:
```suggestion
work_request_link = "
https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0161&prefill_Grant+Name=Chronicle"
```

File: content/grants/app-lexicon.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+[extra]
+image = ""
+description = "A simple dictionary app with ability to add custom definitions for a group."
+reward = "1 star ($3k bonus if done by Assembly)"
+assignee = ""
+id = ""
+champion = "Holium"
+completed = false
+work_request_link = ""
```
Comment (mikeosborne) [inline content/grants/app-lexicon.md]:
```suggestion
work_request_link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0162&prefill_Grant+Name=Lexicon"
```

File: content/grants/app-minesweeper.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+[extra]
+image = ""
+description = "A redesigned version of minesweeper where all of the game logic is written in hoon"
+reward = "1 star ($2k if done by assembly)"
+assignee = ""
+id = ""
+champion = "Holium"
+completed = false
+work_request_link = ""
```
Comment (mikeosborne) [inline content/grants/app-minesweeper.md]:
```suggestion
work_request_link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0163&prefill_Grant+Name=Minesweeper"
```

File: content/grants/app-minesweeper.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "A redesigned version of minesweeper where all of the game logic is written in hoon"
+reward = "1 star ($2k if done by assembly)"
+assignee = ""
+id = ""
```
Comment (mikeosborne) [inline content/grants/app-minesweeper.md]:
```suggestion
grant_id = "B0163"
```

File: content/grants/app-lexicon.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "A simple dictionary app with ability to add custom definitions for a group."
+reward = "1 star ($3k bonus if done by Assembly)"
+assignee = ""
+id = ""
```
Comment (mikeosborne) [inline content/grants/app-lexicon.md]:
```suggestion
grant_id = "B0162"
```

File: content/grants/app-lexicon.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
++++
+title = "Lexicon: simple dictionary app with ability to add custom definitions"
```
Comment (mikeosborne) [inline content/grants/app-lexicon.md]:
```suggestion
title = "Lexicon: simple dictionary app"
```

File: content/grants/app-minesweeper.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
++++
+title = "Minesweeper: Every OS needs it"
```
Comment (mikeosborne) [inline content/grants/app-minesweeper.md]:
```suggestion
title = "Minesweeper"
```

File: content/grants/app-chronicle.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "Chronicle is a news app that links all articles from group channels"
+reward = "2 stars ($4k bonus if done by Assembly)"
+assignee = ""
+id = ""
```
Comment (mikeosborne) [inline content/grants/app-chronicle.md]:
```suggestion
grant_id = "B0161"
```

File: content/grants/app-chronicle.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
++++
+title = "Chronicle: News app that surfaces links from group channels"
```
Comment (mikeosborne) [inline content/grants/app-chronicle.md]:
```suggestion
title = "Chronicle: News app"
```

File: content/grants/app-abacus.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "Abacus: A basic calculator app"
+reward = "1 star ($1k bonus if done by Assembly)"
+assignee = ""
+id = ""
```
Comment (mikeosborne) [inline content/grants/app-abacus.md]:
```suggestion
grant_id = "B0160"
```

File: content/grants/app-abacus.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
++++
+title = "Abacus: basic calculator with ability to convert units"
```
Comment (mikeosborne) [inline content/grants/app-abacus.md]:
```suggestion
title = "Abacus: basic calculator"
```

File: content/grants/app-abacus.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+date = "2022-07-18"
+
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "Abacus: A basic calculator app"
```
Comment (mikeosborne) [inline content/grants/app-abacus.md]:
```suggestion
description = "Abacus: A basic calculator app with the ability to convert units"
```

File: content/grants/app-trove.md [hunk 14]
```diff
@@ -0,0 +1,90 @@
...
+
+[extra]
+image = ""
+description = "A shared file drive allowing the upload of files to an S3 bucket or IPFS and the ability to organize the file links in folders."
+reward = "2 stars ($4k bonus if done by Assembly)"
+assignee = ""
+id = ""
+champion = "Holium"
+completed = false
+work_request_link = ""
```
Comment (mikeosborne) [inline content/grants/app-trove.md]:
```suggestion
work_request_link = "
https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0164&prefill_Grant+Name=Trove"
```

File: content/grants/app-trove.md [hunk 15]
```diff
@@ -0,0 +1,90 @@
...
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "A shared file drive allowing the upload of files to an S3 bucket or IPFS and the ability to organize the file links in folders."
+reward = "2 stars ($4k bonus if done by Assembly)"
+assignee = ""
+id = ""
```
Comment (mikeosborne) [inline content/grants/app-trove.md]:
```suggestion
grant_id = "B0164"
```

File: content/grants/app-trove.md [hunk 16]
```diff
@@ -0,0 +1,90 @@
++++
+title = "Trove: group file drive"
+date = ""
```
Comment (mikeosborne) [inline content/grants/app-trove.md]:
```suggestion
date = "2022-07-18"
```

**Review by mikeosborne — CHANGES_REQUESTED** [review]
Added grant id's and work request links. Shortened some titles so they present better on the grants page.

Also, please remove the "app-" string from the file names so they sort correctly.

**Comment by drunkplato** [comment]
Made all requested changes. Let me know if there is anything else.

**Review by mikeosborne — APPROVED** [review]

**Pull request merged**